QuoteGeneration/installer/linux/rpm/libsgx-ae-qe3/libsgx-ae-qe3.spec

```diff
@@ -60,8 +60,6 @@ done
 
 %files -f %{_specdir}/list-%{name}
 
-%debug_package
```

Comment:
Is debuginfo package necessary in case people want to debug the code?

Reply:
All our AEs have been stripped. That's why in some distros the rpm builder will report an error because it cannot find debug symbol within binary.